src/dataframe/dataframe.jl

```diff
@@ -1024,15 +930,23 @@ select!(df::DataFrame, c::Any) = select!(df, index(df)[c])
 
 """
     select(df::DataFrame, inds, copycols::Bool=true)
+    select(dfv::SubDataFrame, inds, copycols::Bool=true)
+    select(dfr::DataFrameRow, inds)
 
-Create a new `DataFrame` that contains columns from `df` specified by `inds` and return it.
-
-Argument `inds` can be any index that is allowed for column indexing of a `DataFrame`.
-
+If `df` is a `DataFrame` create a new `DataFrame` that contains columns from `df`
+specified by `inds` and return it.
 If `copycols=true` (the default), then returned `DataFrame` holds
 copies of column vectors in `df`.
 If `copycols=false`, then returned `DataFrame` shares column vectors with `df`.
 
+If `select` is called on `SubDataFrame` then
```

Comment:
```suggestion
If `df` is a `SubDataFrame` then
```
(also break next line at 92 chars)